kubectl/plan.sh

```diff
@@ -19,12 +19,20 @@ pkg_build_deps=(
   core/diffutils
   core/which
   core/rsync
+  core/coreutils
 )
 
 pkg_deps=(
   core/glibc
 )
 
+do_prepare() {
+  if [[ ! -r /usr/bin/env ]]; then
```

Comment:
i've seen this pattern so many times now, we should totally make a plan helper for this.  :)

Reply:
I agree - every time I use it I have to go search for it in core-plans to copy/paste :joy: